sql/updates/reverse-dev.sql

```diff
@@ -0,0 +1,2 @@
+-- revert changes to continuous aggregates view definition
+DROP VIEW IF EXISTS timescaledb_information.continuous_aggregates;
```

Comment:
I think in this case we cannot downgrade. If there are compressed materialized hypertables and we downgrade then we break the installation.

Reply:
modified to block downgrade for this case.